### ISSUE TITLE
0.1.0 published with wrong artifactID

### ISSUE THREAD

**Issue by japgolly** [scala-js/scala-js-env-selenium#8]
README says to add

``` scala
libraryDependencies += "org.scala-js" %% "scalajs-env-selenium" % "0.1.0"
```

0.1.0 was published as `seleniumjsenv`, not `scalajs-env-selenium`.

Artefacts: http://repo1.maven.org/maven2/org/scala-js/seleniumjsenv_2.10/0.1.0/

**Comment by sjrd** [comment]
Oops ... my fault! I will fix it today. Thanks for the report!

**Comment by japgolly** [comment]
np :)​

**Comment by japgolly** [comment]
Oh, I nearly forgot: 0.1.0 was published for Scala versions 2.1{0,1,2-M3} where as it's only needed for 2.10 because SBT.

**Comment by sjrd** [comment]
I know but we do publish or entire "tools suite" on all versions, in case people want to build non-sbt things on top.

**Comment by japgolly** [comment]
No worries. Wasn't sure whether it was deliberate or not.

**Comment by sjrd** [comment]
Published 0.1.1 with the correct artifact name. Should hit Maven soon.
